doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/threads-process-con.adoc

```diff
@@ -0,0 +1,26 @@
+[id='threads-process-con_{context}']
+// Stetson: shall we condition this text, or a part of it, only for community documentation?
+= Threads in the {PROCESS_ENGINE}
+
+We can refer to two types of "multi-threading":  _logical_ and _technical_. _Technical multi-threading_ involves multiple threads or processes that are started, for example, by a Java or  C program. _Logical multi-threading_ happens in a BPM process, for example, after the process reaches a parallel gateway. In execution logic, the original process splits into two processes that run in a parallel fashion.
```

Comment:
Informal and unclear, suggest something like, "Applications typically implement two general types of multi-threading: ..."  Also note again that we shouldn't be using quotes for emphasis in docs.